app/assets/v2/js/pages/tribe-edit.js

```diff
@@ -0,0 +1,89 @@
+let quill;
+
+$('#edit-btn').on('click', function() {
+  const activateQuill = () => {
+    if (quill) {
+      return quill.isEnabled() ? destroyQuill() : rebuildQuill();
+    }
```

Comment:
we could rewrite this as 

`return quiill && quill.isEnabled() ? destroyQuill() : rebuildQuill();`

Reply:
quill is undefined in the first run that will block the code to continue

Reply:
derp derp ! you are right